{
######### Common ##########




######## Captain ##########

default_desired_speed   :  1,
default_m_pt_radius     :  3,
mission_file_location   :  /home/odroid/Missions/mission.txt,  
#home location, lat lon takes precedence
home_lat      :  39.449768,
home_lon      : -76.5519567,
homeX          : 0,
homeY          : 0,




######## Vehicle_Simulated ###############

turnRate    : 30, #deg per second
thrustScale : 1,  #scale of thrust to speed conversion

#ocean current simulation 
currentSpeed    : 0, #meter per second
currentYaw      : 0, #radian

startX          : 0,
startY          : 0,


######## Params for lawnmow agent ##########
lm_xlength      : 50,         #meters
lm_ylength      : 50,         #meters
lm_mowewidth    : 20,         #meters
lm_mowebearing  : 20           #deg




}
